release-team/role-handbooks/docs/README.md

```diff
@@ -74,12 +106,102 @@ In addition to the time requirements above, a Docs Lead must:
 
 Docs Lead Shadows are people who are preparing to be a Docs Lead in the future. In addition to the time requirements above, shadows must:
 
+- Strong written and verbal communications skills
+- A working knowledge of Kubernetes concepts
+- Project management experience is helpful but not required
 - Have signed the [contributor CLA](https://github.com/kubernetes/community/blob/master/CLA.md) for Kubernetes.
 - Be invested in becoming an org member within the release cycle. This can often be achieved during the release cycle with sponsorship from a role lead. See the [Release Team onboarding guide](/release-team/release-team-onboarding.md) for more details.
-- General knowledge of our SIG-Docs [areas of responsibility](https://github.com/kubernetes/community/tree/master/sig-docs#subprojects).
+- General knowledge of our SIG Docs [areas of responsibility](https://github.com/kubernetes/community/tree/master/sig-docs#subprojects).
 - Experience with the general process involved with [contributing](https://kubernetes.io/docs/contribute/start/) to Kubernetes website.
 - Be a [milestone maintainer](https://github.com/orgs/kubernetes/teams/website-milestone-maintainers/) in order to have the ability to add a milestone to an issue. Access can be requested by creating a [PR](https://github.com/kubernetes/org/pull/2235) against `kubernetes/org` repo.
 
+### Machine and GitHub Setup
+
+#### Setup krel
+
+[Install Go](https://golang.org/doc/install) in your machine and follow the [instructions to build the release tools](https://github.com/kubernetes/release/tree/master/docs/krel#installation) in your machine. Check the system requirements in the krel documentation.
```

Comment:
We should also mention that`jq` is a hard requirement for running the `krel` command. This is not documented anywhere, and my fellow shadows encountered issues simply because they didn’t have `jq` installed.
@satyampsoni @npolshakova

Reply:
@rayandas did you ever track down what the error was?

Reply:
@npolshakova I didn’t. But I will try to regenerate it today. And let you know.

Reply:
I couldn’t regenerate it on my system. I will keep an eye on errors faced by new contributors around this and then report it with details.